tests/queries/0_stateless/03228_async_insert_query_params.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+# Tags: no-parallel
```

Comment:
FWIW, 9 out of 40 async_insert tests have the `no-parallel` tag
```
$ find tests/queries/ -name '*async_insert*[sh|sql]' 2>/dev/null | xargs grep no-parallel
tests/queries/0_stateless/02884_async_insert_skip_settings.sql:-- Tags: no-parallel
tests/queries/0_stateless/02726_async_insert_flush_queue.sql:-- Tags: no-parallel
tests/queries/0_stateless/02884_async_insert_native_protocol_1.sh:# Tags: no-parallel
tests/queries/0_stateless/02884_async_insert_native_protocol_3.sh:# Tags: no-parallel
tests/queries/0_stateless/02246_async_insert_quota.sh:# Tags: no-parallel, no-fasttest
tests/queries/0_stateless/02481_async_insert_dedup.sh:# Tags: long, zookeeper, no-parallel, no-fasttest
tests/queries/0_stateless/02481_async_insert_dedup_token.sh:# Tags: long, zookeeper, no-parallel, no-fasttest
tests/queries/0_stateless/02515_cleanup_async_insert_block_ids.sh:# Tags: zookeeper, no-parallel, long, no-fasttest, no-replicated-database
$ find tests/queries/ -name '*async_insert*[sh|sql]' 2>/dev/null | xargs grep no-parallel | wc -l
9
$ find tests/queries/ -name '*async_insert*[sh|sql]' 2>/dev/null | wc -l
40
```

Reply:
This test uses `SYSTEM FLUSH ASYNC INSERT QUEUE` query which flushes the global queue. So it can affect other tests and make them flaky. I decided to make it non-parallel from the beginning to avoid further debug of flaky tests.